apisix/plugins/grpc-transcode/proto.lua

```diff
@@ -270,5 +270,10 @@ function _M.init()
     end
 end
 
+function _M.destroy()
+    if protos then
+        protos:close()
```

Comment:
Could we add a test to ensure the `close` is called? For instance, via injecting some code.